.github/workflows/main.yml

```diff
@@ -17,3 +17,9 @@ jobs:
           otp-version: ${{ matrix.otp }}
       - run: |
           make
+      - name: Archive CT Logs
+        uses: actions/upload-artifact@v2
+        with:
+          name: ct-logs
+          path: _build/test/
```

Comment:
```suggestion
          path: _build/test/logs/
```
?

Reply:
reserve it for cover log as well. I will leave it here.

Reply:
I think it'll upload the `lib` sub dir as well.  But it might not be a problem, since `lc` is small.